### PR TITLE
Update Units.json

### PR DESCRIPTION
**Pull request by SchoeniosX**
Religion changes
- Religious Units
- "Great Prophet"

### CODE REVIEW

File: jsons/Units.json
Comment (SchoeniosX) [inline jsons/Units.json:1]: Supposed to be working

**Review by Mr-Chubz — APPROVED** [review]

**Comment by SchoeniosX** [comment]
Hello Chubz,


Would this Acropolis.png looks prettier than the current one?

Mr-Chubz ***@***.***> schrieb am Mi., 26. Juni 2024, 01:59:

**Comment by Mr-Chubz** [comment]
Hello Lauren S,

I'm struggling to find the Acropolis.png you mentioned.

I appreciate you patching Warfare Edited and I hope you are well.

Sincerely,
Chubz

**Comment by SchoeniosX** [comment]
Hello Chubz,

The exact file is called: "game.png" should be right below the email text I
wrote. The image should be intended for "Uniques From Chubz" mod. There I
made a pull request to change the Acropolis. And yes thank you, I'm fine?
And you?

Sincerely Laurin S.

Mr-Chubz ***@***.***> schrieb am Do., 27. Juni 2024, 00:52:

> Hello Lauren S,
>
> I'm struggling to find the Acropolis.png you mentioned.
>
> I appreciate you patching Warfare Edited and I hope you are well.
>
> Sincerely,
> Chubz
>

**Comment by SchoeniosX** [comment]
Laurin Schönwälder ***@***.***> schrieb am Do., 27. Juni
2024, 01:02:

> Hello Chubz,
>
> The exact file is called: "game.png" should be right below the email text
> I wrote. The image should be intended for "Uniques From Chubz" mod. There I
> made a pull request to change the Acropolis. And yes thank you, I'm fine?
> And you?
>
> Sincerely Laurin S.
>
> Mr-Chubz ***@***.***> schrieb am Do., 27. Juni 2024, 00:52:
>
>> Hello Lauren S,
>>
>> I'm struggling to find the Acropolis.png you mentioned.
>>
>> I appreciate you patching Warfare Edited and I hope you are well.
>>
>> Sincerely,
>> Chubz
>>

**Comment by Mr-Chubz** [comment]
I'm well. Thank you. I don't see any attachments or links attached to the
email. Forgive me if I'm oblivious. I'm not tech-savvy.

**Comment by SchoeniosX** [comment]
Hello Chubz,

In your mod "Uniques From Chubz" there should be an image (game.png) in my
pull request in the comments. There you can have the link directly by
clicking on the picture.

Sincerely Laurin S.




Mr-Chubz ***@***.***> schrieb am Do., 27. Juni 2024, 01:22:

> I'm well. Thank you. I don't see any attachments or links attached to the
> email. Forgive me if I'm oblivious. I'm not tech-savvy.
>

**Comment by Mr-Chubz** [comment]
It won't allow me to save the PNG as it is private.

I haven't touched this repository in over a year. I'm inviting you to
collaborate so that you do as you wish with it.

**Comment by SchoeniosX** [comment]
Hello Chubz,

I thank you very much for this invitation to collaborate. Wow, that
surprises me a lot. I'm happy to accept it and see what I can do. I'm
honored, thank you!

Sincerely Laurin Schönwälder

Mr-Chubz ***@***.***> schrieb am Do., 27. Juni 2024, 01:56:

> It won't allow me to save the PNG as it is private.
>
> I haven't touched this repository in over a year. I'm inviting you to
> collaborate so that you do as you wish with it.
>

**Comment by Mr-Chubz** [comment]
You're welcome! I'm glad it's something you're interested in doing. The
general idea is that it's compatible with Warfare Edited. (The only thing
making them incompatible is the unit types). It would thus also be
compatible with The Ultimate Modpack. I think Warfare Edited is behind TUM2
in some places.

**Pull request merged**